compiler/ballerina-backend-jvm-old/src/main/ballerina/src/compiler_backend_jvm/interop/external_method_gen.bal

```diff
@@ -192,6 +192,12 @@ function createOldStyleExternalFunctionWrapper(bir:Function birFunc, string orgN
     addDefaultableBooleanVarsToSignature(birFunc);
     bir:BInvokableType functionTypeDesc = birFunc.typeValue;
 
+    bir:BType? restType = functionTypeDesc?.restType;
+
+    if (!(restType is ())) {
+        jMethodPramTypes[jMethodPramTypes.length()] = restType;
```

Comment:
Can't we do
```suggestion
        jMethodPramTypes.push(restType);
```

Applicable elsewhere too.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/19976/commits/7432217a84fd35c7ae7682d1e9173e9416f9ccdc